test/uk/gov/hmrc/cdsimportsddsfrontend/services/CustomsDeclarationsServiceSpec.scala

```diff
@@ -74,7 +75,25 @@ class CustomsDeclarationsServiceSpec extends WordSpec
 
       response.conversationId mustBe decApiResponse.conversationId
       response.status mustBe decApiResponse.status
-      response.xml mustBe "&lt;DeclaringMyStuff/&gt;"
+      response.xml mustBe DeclarationXml.prettyPrintToHtml(decWithLRN)
+
+      verify(mockHttp, times(1)).POSTString(any(), any(), any())(any(), any(), any())
```

Comment:
can we be more specific here?  expected URL, body, headers, whatever?

Reply:
updated tests for the same